src/manage/workspaces/index.md

```diff
@@ -50,32 +50,34 @@ From the list of available workspaces, select the one you want to switch to:
 ## How to create a new workspace
 1. Log in to <AppDomainNameLink />
 2. Click on your account and navigate to **Manage workspaces**
-   ![Manage workspaces in Mergin Maps web dashboard](./web-manage-workspaces.jpg "Manage workspaces in Mergin Maps web dashboard")
-3. Here, you can see the overview of your existing workspaces. Click on **Create workspace**
-   ![Create workspace in Mergin Maps web dashboard](./web-create-workspace.jpg "Create workspace in Mergin Maps web dashboard")
+   ![Manage workspaces in Mergin Maps web dashboard](./dashboard-ov-manage-workspaces.jpg "Manage workspaces in Mergin Maps web dashboard")
+3. Here, you can see the overview of your workspaces. Click on **Create workspace**
+   ![Create workspace in Mergin Maps web dashboard](./dashboard-create-workspace.jpg "Create workspace in Mergin Maps web dashboard")
 4. Fill in the name and description of your new workspace
-   ![Create workspace in Mergin Maps web dashboard details](./web-new-workspace.jpg "Create workspace in Mergin Maps web dashboard details")
+   ![Create workspace in Mergin Maps web dashboard details](./dashboard-new-workspace.jpg "Create workspace in Mergin Maps web dashboard details")
    
    :::warning
    The name of a workspace cannot be changed later.
    :::
    
 5. Now you have a new workspace! You can [create projects](../create-project/) in the workspace and [invite other users](../project-advanced/#add-users-to-a-workspace) to contribute.
-   ![Mergin Maps web dashboard workspaces overview](./web-new-workspace-created.jpg "Mergin Maps web dashboard workspaces overview")
+   ![Mergin Maps web dashboard workspaces overview](./dashboard-new-workspace-created.jpg "Mergin Maps web dashboard workspaces overview")
 
-Currently, it is possible to have up to 3 workspaces in the trial plan (without subscription). Note that every workspace has its own subscription. When created, the workspace starts with the free trial. After the trial period, you will have to [switch to a paid subscription](../../manage/subscriptions/#how-to-change-a-subscription). 
+Note that every workspace has its own subscription. When created, the workspace starts with the free trial. After the trial period, you will have to [switch to a paid subscription](../../manage/subscriptions/#how-to-change-a-subscription). Currently, it is possible to have up to 3 workspaces without subscription (using the trial) simultaneously. 
```

Comment:
Edit text:
Note that every workspace has its own subscription. When created, the workspace starts with a free trial. After the trial period, you will have to [switch to a paid subscription](../../manage/subscriptions/#how-to-change-a-subscription). Currently, it is possible to have up to 3 workspaces without a subscription (using the trial) simultaneously.